python/kserve/kserve/protocol/infer_type.py

```diff
@@ -630,4 +646,44 @@ def to_grpc(self) -> ModelInferResponse:
             infer_outputs.append(infer_output_dict)
 
         return ModelInferResponse(id=self.id, model_name=self.model_name, outputs=infer_outputs,
-                                  raw_output_contents=raw_output_contents)
+                                  raw_output_contents=raw_output_contents,
+                                  parameters=to_grpc_parameters(self.parameters))
+
+
+def to_grpc_parameters(parameters: Union[dict, MessageMap[str, InferParameter]]) -> Dict[str, InferParameter]:
+    """
+    Converts REST parameters to GRPC InferParameter objects
+
+    :param parameters: parameters to be converted.
+    :return: converted parameters as Dict[str, InferParameter]
+    :raises InvalidInput: if the parameter type is not supported.
+    """
+    if isinstance(parameters, dict):
+        for key, val in parameters.items():
+            if isinstance(val, str):
+                parameters[key] = InferParameter(string_param=val)
+            elif isinstance(val, bool):
+                parameters[key] = InferParameter(bool_param=val)
+            elif isinstance(val, int):
```

Comment:
We do not support float ?

Reply:
Yes, we don't support float. https://github.com/kserve/kserve/blob/c254e704c3719d3310391ee994123d5b49588e5d/python/kserve/kserve/protocol/grpc/grpc_predict_v2.proto#L269C1-L284C2